nearcore/src/config.rs

```diff
@@ -1077,6 +1096,30 @@ pub fn download_genesis(url: &String, path: &PathBuf) {
             .await
             .expect("Genesis file is bigger than 10GB. Please make the limit higher.");
 
+        std::fs::write(&path, &body).expect("Failed to create / write a genesis file.");
+
+        info!(target: "near", "Saved the genesis file to: {} ...", path.as_path().display());
+    });
+}
+
+pub fn download_config(url: &String, path: &PathBuf) {
```

Comment:
I suggest adding (extracting) the download-file helper function instead of copy-pasting the code.